src/System Application/App/Email/src/Connector/DefaultEmailConnectorv2.Codeunit.al

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Email;
+
+codeunit 8998 "Default Email Connector v2" implements "Email Connector v2"
```

Comment:
What is the purpose of this codeunit?

Reply:
To prevent the enum implementation of the connector from breaking existing extensions on the enum. I'd break all our partners due to the new implementation on the enum without this.